EIPS/eip-7702.md

```diff
@@ -90,6 +90,8 @@ The delegation designation uses the banned opcode `0xef` from [EIP-3541](./eip-3
 
 For example, `EXTCODESIZE` would return `2` (the size of `0xef01`) instead of `23` which would represent the delegation designation, `EXTCODEHASH` would return `0xeadcdba66a79ab5dce91622d1d75c8cff5cff0b96944c3bf1072cd08ce018329` (`keccak256(0xef01)`), and `CALL` would load the code from `address` and execute it in the context of `authority`.
 
+`CODESIZE` and `CODECOPY` instructions operate on executable code, as before. *Note that in a delegated execution `CODESIZE` and `CODECOPY` produce different result comparing to `EXTCODESIZE` and `EXTCODECOPY` of execution target.*
```

Comment:
I did not realize this before, but this is the first time we have a case where `CODECOPY`/`CODESIZE` vs `EXTCODECOPY`/`EXTCODESIZE` on itself (`ADDRESS`) returns something different, right? (Except in the context of a contract creation frame)

Reply:
Yes, but it also answers the question "Am I being run as a delegate" - which is a useful security feature.

Reply:
I don't think this is true. CODECOPY copies the code that is dynamically running whereas EXTCODECOPY on ADDRESS copies the account code, these can already differ prior to EIP-7702 due to DELEGATECALL or CALLCODE.